recipes/bagit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+# https://github.com/LibraryOfCongress/bagit-python/issues/70
+{% set name = "bagit" %}
+{% set version = "1.5.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/LibraryOfCongress/bagit-python.git
+  git_rev: 613a82849d0dd43761e005606cd4c7ba3cd44188
```

Comment:
@kwilcox please add `{% set commit = "613a82849d0dd43761e005606cd4c7ba3cd44188" %}` at the top and then change these to:

```yaml
  fn: {{ commit }}.tar.gz
  url: https://github.com/LibraryOfCongress/bagit-python/archive/{{ commit }}.tar.gz
  sha256: d43091a9ce5915d667d4af5eb11345dadadda2d5b084fd2cfdeb2d03473a84fd
```